src/System.Drawing.Common/src/System/Drawing/Font.cs

```diff
@@ -14,6 +14,7 @@ namespace System.Drawing
     /// <summary>
     /// Defines a particular format for text, including font face, size, and style attributes.
     /// </summary>
+    [TypeConverter("System.Drawing.FontConverter, System.Windows.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51")]
```

Comment:
The reason why you're getting the failures in the AllConfigurations leg is because TypeConverterAttribute was in `System.ComponentModel.TypeConverters` in netcoreapp2.0 and in netcoreapp3.0 is in `System.ObjetModel` which you added a reference for. However these converters are not interesting for netcoreapp2.0 since we're adding them for winforms support which only comes into play in netcoreapp3.0. Also System.Windows.Extensions is only supported starting in 3.0.

The fix is to #if def this TypeConverters to only be included in netcoreapp.

Reply:
i.e 
```cs
#if netcoreapp
[TypeConverter.....]
#endif
```

netcoreapp constant is already defined for all projects when they target netcoreapp.